installer/lib/mix/tasks/phx.new.ex

```diff
@@ -40,6 +40,8 @@ defmodule Mix.Tasks.Phx.New do
 
     * `--no-gettext` - do not generate gettext files.
 
+    * `--live` - include Phoenix LiveView.
```

Comment:
```suggestion
    * `--live` - include Phoenix LiveView to make it easier than ever
      to build interactive, real-time applications
```

Also, please move this option up in the docs.

With love,

The Marketing Department.